src/common/enums/alkemio.error.status.ts

```diff
@@ -28,4 +28,5 @@ export enum AlkemioErrorStatus {
   MATRIX_AGENT_POOL_ERROR = 'MATRIX_AGENT_POOL_ERROR',
   MATRIX_ENTITY_NOT_FOUND_ERROR = 'MATRIX_ENTITY_NOT_FOUND_ERROR',
   BOOTSTRAP_FAILED = 'BOOTSTRAP_FAILED',
+  SESSION_UNDEFINED = 'COMMUNICATION_SESSION_UNDEFINED',
```

Comment:
SESSION_UNDEFINED is a bit of a misleading name as it implies you are talking about a session in the context of communications, but we could also have kratos undefined session and in the context of the application this might be misleading - change to either COMMUNICATION_SESSION_UNDEFINED or MATRIX_SESSION_UNDEFINED.

Reply:
COMMUNICATIONS_SESSION_UNDEFINED - like the value